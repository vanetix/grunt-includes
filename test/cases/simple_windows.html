<html>
  <head>
    <title>Includes test</title>
  </head>
  <body>
    <div class="header">
      include "header_windows.html"
    </div>

    <div class="body">
      <p>Hello world</p>
    </div>

    <div class="footer">
      include "footer_windows.html"
    </div>
  </body>
</html>
