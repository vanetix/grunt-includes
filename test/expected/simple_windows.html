<html>
  <head>
    <title>Includes test</title>
  </head>
  <body>
    <div class="header">
<header>
  <h1>Header</h1>
</header>

    </div>

    <div class="body">
      <p>Hello world</p>
    </div>

    <div class="footer">
<footer>
  <h1>Footer</h1>
</footer>

    </div>
  </body>
</html>
